e2e-tests/trade-finance-manager/cypress/integration/journeys/case/amendments/amendment-should-change-all-facilities-values.spec.js

```diff
@@ -0,0 +1,141 @@
+  let dealId;
+  const dealFacilities = [];
+
+  before(() => {
+    cy.insertOneDeal(MOCK_DEAL_AIN, MOCK_MAKER_TFM).then((insertedDeal) => {
+      dealId = insertedDeal._id;
+      const { dealType, mockFacilities } = MOCK_DEAL_AIN;
+      cy.createFacilities(dealId, [mockFacilities[0]], MOCK_MAKER_TFM).then((createdFacilities) => {
+        dealFacilities.push(...createdFacilities);
+      });
+
+      cy.submitDeal(dealId, dealType);
+    });
+  });
+
+  after(() => {
+    cy.deleteDeals(dealId, ADMIN_LOGIN);
+    dealFacilities.forEach((facility) => {
+      cy.deleteFacility(facility._id, MOCK_MAKER_TFM);
+    });
+  });
+
+  it('should show facility value and coverEndDate for original facility', () => {
+    cy.login(PIM_USER_1);
+    cy.visit(relative('/facilities'));
+    cy.url().should('eq', relative('/facilities'));
+
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').last().as('row1');
+    cy.get('@row1').find('[data-cy="facility__ukefFacilityId"]').should('contain', '1000000');
+    cy.get('@row1').find('[data-cy="facility__facilityValue"]').should('contain', 'GBP 12,345');
+    cy.get('@row1').find('[data-cy="facility__coverEndDate"]').should('contain', '20 Oct 2022');
+  });
+
+  it('should submit an automatic amendment request for coverEndDate', () => {
+    cy.login(PIM_USER_1);
+    const facilityId = dealFacilities[0]._id;
+    cy.visit(relative(`/case/${dealId}/facility/${facilityId}`));
+
+    facilityPage.facilityTabAmendments().click();
+    amendmentsPage.addAmendmentButton().should('exist');
+    amendmentsPage.addAmendmentButton().contains('Add an amendment request');
+    amendmentsPage.addAmendmentButton().click();
+    cy.url().should('contain', 'request-date');
+
+    amendmentsPage.amendmentRequestDayInput().clear().focused().type(dateConstants.todayDay);
+    amendmentsPage.amendmentRequestMonthInput().clear().focused().type(dateConstants.todayMonth);
+    amendmentsPage.amendmentRequestYearInput().clear().focused().type(dateConstants.todayYear);
+    amendmentsPage.continueAmendment().click();
+    cy.url().should('contain', 'request-approval');
+
+    // automatic approval
+    amendmentsPage.amendmentRequestApprovalNo().click();
+    amendmentsPage.continueAmendment().click();
+    cy.url().should('contain', 'amendment-effective-date');
+
+    amendmentsPage.amendmentEffectiveDayInput().clear().focused().type(dateConstants.todayDay);
+    amendmentsPage.amendmentEffectiveMonthInput().clear().focused().type(dateConstants.todayMonth);
+    amendmentsPage.amendmentEffectiveYearInput().clear().focused().type(dateConstants.todayYear);
+    amendmentsPage.continueAmendment().click();
+    cy.url().should('contain', 'amendment-options');
+
+    amendmentsPage.amendmentCoverEndDateCheckbox().should('not.be.checked');
+    amendmentsPage.amendmentFacilityValueCheckbox().should('not.be.checked');
+    // update both the cover end date and the facility value
```

Comment:
It looks like you are only updating the cover end date here and not the facility value, can we reflect this in the comment please.

Reply:
done